src/parquet/file/metadata.h

```diff
@@ -92,7 +82,7 @@ class ApplicationVersion {
   bool VersionEq(const ApplicationVersion& other_version) const;
 
   // Checks if the Version has the correct statistics for a given column
-  bool HasCorrectStatistics(Type::type primitive) const;
+  bool HasCorrectStatistics(Type::type primitive, SortOrder::type sort_order) const;
```

Comment:
As we change the interface here, we should make the new version 1.3.0. 

@majetideepak alternatively can we provide a default argument here?